src/mesh/SX128xInterface.cpp

```diff
@@ -25,7 +25,12 @@ bool SX128xInterface<T>::init()
     digitalWrite(SX128X_POWER_EN, HIGH);
     pinMode(SX128X_POWER_EN, OUTPUT);
 #endif
-
+  
+#ifdef RF95_FAN_EN
+    pinMode(RF95_FAN_EN, OUTPUT);
+    digitalWrite(RF95_FAN_EN, 1);
+#endif
```

Comment:
Given it has high tramit power ability it might make sense for now to start the fan and keep it running. We could see how we could change it it future. There iscanother ELRS with oled that has only heatsink

Reply:
There is an option to rotate the fan direction and on the ExpressELRS site showing temperatures

Reply:
okay, then let's leave it like this for now :-)